person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
+  def initialize(id, name, age, parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def of_age?
+    if @age >= 18
+      puts true
+    else
+      puts false
+    end
+  end
```

Comment:
- [ ] Kindly make sure this method is private by adding `private` before it and this method should return true or false not print, so it will be like this:
```ruby
  def is_of_age?
    @age >= 18
  end
```

Reply:
@omarsalem7 Allright! Thanks for your review!